numba/core/typing/builtins.py

```diff
@@ -654,6 +654,26 @@ def generic(self, args, kws):
             sig = signature(ret, *args)
             return sig
 
+@infer
+class StaticGetItemClass(AbstractTemplate):
+    """This handles the "static_getitem" that occurs when a Numba type has a
+    __getitem__ made on it in input source e.g:
```

Comment:
I found this a bit hard to parse and understand - is this simplification clearer and easier?

```suggestion
    """This handles the "static_getitem" when a Numba type is subscripted,
    e.g.:
```